dev-environment/h2o/models/glm-wine/train.py

```diff
@@ -11,18 +11,18 @@
 model_id = int(sys.argv[2])
 
 h2o_port = int(os.environ['H2O_SERVER_PORT'])
-h2o.init(port = h2o_port)
+h2o.init(port=h2o_port)
 
-csv_url = ("https://raw.githubusercontent.com/zygmuntz/wine-quality/master/winequality/winequality-red.csv")
```

Comment:
please move this data to our repository (like https://github.com/prinz-nussknacker/banksim1) not to deepnd on others datasources